### DISCUSSION TITLE
¿Podéis asignaros las issues?

### DISCUSSION THREAD

**Discussion by ArisGuimera** [ArisGuimera/JetpackComposePro#42] [Q&A]
Muchos me habéis escrito diciendo que os asigne la tarea y no sé si es que por defecto está bloqueado y tengo que hacerlo yo (el asignar). ¿Alguien me puede confirmar?

**Comment by thonyguev** [comment]
Hola @ArisGuimera, no me sale la opción de asignarme.

**Reply by ArisGuimera** [reply]
Vale, he visto que los tengo que ir asignando yo. El que quiera alguno que deje un comentario en la issue y se lo asigno
